docs/admin/maintenance/noble_migration_prep.rst

```diff
@@ -116,6 +116,24 @@ Please run:
 and send the output to :ref:`Support <getting_support>` immediately, so we can diagnose
 the severity.
 
+Ethernet
+--------
+
+If this fails, it means that you have an ethernet interface that is not connected to a network.
+
+Double-check that your ethernet cable is connected to the correct port on your server. If you are not using recommended hardware, you may have multiple ethernet interfaces configured. In that event, we recommend disabling all but the one you are using for SecureDrop.
+
+In Ubuntu Server, this can be done by removing the additional interfaces in the ``00-installer-config.yaml`` configuration file. For example, you can run:
```

Comment:
It's mentioned below, but let's be explicit on first reference:

```suggestion
In Ubuntu Server, this can be done by removing the additional interfaces in the ``/etc/netplan/00-installer-config.yaml`` configuration file. For example, you can run:
```